src/gsi/unfill_mass_grid2.f90

```diff
@@ -381,3 +381,107 @@ subroutine unfill_mass_grid2t_drycheck(gout,nx,ny,gin,qs)
   end do
   
 end subroutine unfill_mass_grid2t_drycheck
+
+subroutine unfill_mass_grid2t_ldlkmask(gout,nx,ny,gin,landmask, &
+                            lakemask,i_howv_mask)
+!$$$  subprogram documentation block
+!                .      .    .                                       .
+! subprogram:    unfill_mass_grid2t        opposite of fill_mass_grid2
```

Comment:
This subroutine is `unfill_mass_grid2t_ldlkmask`, not `unfill_mass_grid2t`

Reply:
@RussTreadon-NOAA 
Hi Russ,

I updated this information in documentation block. Thank you!